package.json

```diff
@@ -23,7 +23,7 @@
     "bcryptjs": "^2.4.3",
     "body-parser": "^1.18.3",
     "compression": "^1.7.3",
-    "connect-mongo": "^2.0.3",
+    "connect-mongo": "^3.2.0",
```

Comment:
not needed.
All packages related to the API should be removed since they are not used here any more